libsweep/src/unix/serial.cc

```diff
@@ -26,6 +26,15 @@ static speed_t get_baud(int32_t bitrate) {
     throw error{"Only baud rate 115200 is supported at this time."};
     return -1;
   }
+
+  // translate human readable bitrate to termios bitrate
+  if (bitrate == 115200)
```

Comment:
This conditional branch is redundant given the `if (bitrate != 115200)` above, which results in an error. Maybe just 
```c++
// if defined, translate human readable bitrate to termios bitrate
#ifdef B115200
    return B115200
#endif
return bitrate;
```